### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Bankar-Swapnil**
Project setup

### CODE REVIEW

File: src/app/body/body.component.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.hello{
```
Comment (viv9k) [inline src/app/body/body.component.css]: hello?

**Review by viv9k — CHANGES_REQUESTED** [review]
Let's discuss on these changes as I'm not sure if it's good

File: src/app/body/body.component.css (hunk 1 above)
Comment (Bankar-Swapnil) [reply src/app/body/body.component.css]: just for  temporary CSS class

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**